classes/form/CustomerForm.php

```diff
@@ -275,4 +277,46 @@ private function validateByModules()
             }
         }
     }
+
+    /**
+     * Performs validation on field values.
+     * Adds error to the field object if value is not as expected.
+     */
+    private function validateFieldsValues(): void
+    {
+        $this->validateFirstname();
+        $this->validateLastname();
+    }
+
+    private function validateFirstname(): void
```

Comment:
This function seems almost identical to `validateLastname()` - only field name changes. Maybe we could have a `validateFiedIsCustomerName($fieldName);` unique function for both?

Reply:
:+1: